tests/containers/buildah.pm

```diff
@@ -37,7 +37,7 @@ sub run {
         zypper_call('install skopeo');
 
         # temporarily necessary due to https://bugzilla.suse.com/show_bug.cgi?id=1220568
-        zypper_call('install cni-plugins') if (is_sle("=15-SP3"));
+        zypper_call('install cni-plugins') if (is_sle("15-SP3+"));
```

Comment:
Is it possible any case here, where netwark and aadavark should be considered too?

Reply:
It fails in docker test suite.